src/IoC/ServiceModule.cs

```diff
@@ -146,6 +147,8 @@ protected override void Load(ContainerBuilder builder)
             builder.RegisterType<TqmsJobrefsFacadeService>().As<IFacadeService<TqmsJobRef, string, TqmsJobRefResource, TqmsJobRefResource>>();
             builder.RegisterType<ConsignmentShipfileFacadeService>().As<IConsignmentShipfileFacadeService>();
             builder.RegisterType<ConsignmentFacadeService>().As<IFacadeService<Consignment, int, ConsignmentResource, ConsignmentResource>>();
+            builder.RegisterType<CurrencyFacadeService>()
+                .As<IFacadeService<Currency, string, CurrencyResource, CurrencyResource>>();
```

Comment:
i take it the ImportBookCpc facade and repo already exist in ioc?

Reply:
repo did in persistence but had forgotten to register the facade